### ISSUE TITLE
Make Klipper/Moonraker Log folder tmpfs ?

### ISSUE THREAD

**Issue by LucidityCrash** [Rat-OS/RatOS#10]
Klipper and Moonraker are very chatty even in their quietest modes, I've made the folder they log to a tmpfs file system to reduce SD card wear.  
It can be a pain if your pi crashes or you reboot/power it off before getting what you need,  its a trade off.

**Comment by miklschmidt** [comment]
@LucidityCrash I agree, i think it's a good idea, @worksasintended mentioned this as well. I'm scheduling this for v1.1.0.

**Comment by miklschmidt** [comment]
Moved to 1.2.0
